pkg/machines/libvirt-common.es6

```diff
@@ -566,6 +570,24 @@ export function updateVCPUSettings(domXml, count, max, sockets, cores, threads)
     return tmp.innerHTML;
 }
 
+/*
```

Comment:
As we discussed please use the setMemory API to update current memory, for both live or persistent configuration. And the the defineXML  API to update the max memory.
I am aware that if the is numa configured, the API will not work, but let's not deal with it for now, libvirt will return a nice error to show to the user in this case. @Antique are we aligned here?

Reply:
For now sounds good. I don't want to block this feature and we can change the internal implementation when needed.

Reply:
Thanks.